algorithms/.idea/workspace.xml

```diff
@@ -1,119 +1,49 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would take a stab and put `.idea` directory in `.gitignore` since it's very dev specific file. What do you think ?

Reply:
agreed . thats a very valid point . a very silly one . will correct this.